order/buyer/issue/README.md

```diff
@@ -104,7 +104,11 @@ The optional buyer, buyer accounting or supplier company party:
 * `supplierItemNumber`: the item code or number as known at the supplier. Required in case of wholesale suppliers.
 
 {% hint style="warning" %}
-`item.number` should be unique within your company and never change. Never renumber or re-use `item.number`s.
+`item.number` and `item.supplierItemNumber` are optional and may be left empty, for example in case of service or RFQ orders or text lines.
```

Comment:
"or text lines." might confuse customers. Some ERPs, like BC and Excact have actual text lines, which don't need to be confirmed. In fact they can be compared with TC order/line notes.
Ps in the future we could add actual "text lines" which don't need to be confirmed to mimic this ERP behavior

Reply:
I will remove "or text lines."

> which don't need to be confirmed.

It depends, Damen uses text lines for service orders that have to be confirmed.
But we can add some indicator when the line does not need to be confirmed.